plugins/generator-1.17.1/forge-1.17.1/procedures/world_entity_inrange_foreach.java.ftl

```diff
@@ -0,0 +1,10 @@
+{
+	final Vec3 _center = new Vec3(${input$x}, ${input$y}, ${input$z});
+	List<Entity> _entfound = world.getEntitiesOfClass(Entity.class, new AABB(_center, _center).inflate(${input$range} / 2d), e -> true)

```

Comment:
This AABB approach differes from the ones in normal inrange and inrange exists, I suggest you use the same one for all 3 procedure blocks

Reply:
This one makes it so I don't have to repeat the range input 3 times. I couldn't use it in the other blocks since those are output blocks

Reply:
I see :)